module/os/freebsd/zfs/zvol_os.c

```diff
@@ -544,7 +546,7 @@ zvol_geom_worker(void *arg)
 			continue;
 		}
 		mtx_unlock(&zsg->zsg_queue_mtx);
-		zvol_geom_bio_strategy(bp);
+		zvol_geom_bio_strategy(bp, B_FALSE);
```

Comment:
With introduction of taskq(s), why do we we still have this `zvol_geom_worker()` thread?  Seems we first queue requests to it, and then immediately re-queue again to the taskq.  Am I missing something?

Reply:
You are right. I missed it, the zvol_geom_worker() should be removed.